### ISSUE TITLE
Floating lines on Overhang Walls with Inner/Outer/Inner sequence!

### ISSUE THREAD

**Issue by Vitorleoron** [SoftFever/OrcaSlicer#4709]
### Is there an existing issue for this problem?

- [X] I have searched the existing issues

### OrcaSlicer Version

2.0.0-rc

### Operating System (OS)

Windows

### OS Version

11

### Printer

K1 Max

### How to reproduce

When I printed some object that have some overhang walls, with this settings above:

<img width="407" alt="Configs Orca" src="https://github.com/SoftFever/OrcaSlicer/assets/15314940/86ca2db0-5d97-48df-8ea9-8fa941aa0678">

When it reaches the layer that start close the hole, it prints the Inner wall of the bridge/overhang wall first then the "Outer" and the Inner, so it prints this lines floating, as the video above:

https://github.com/SoftFever/OrcaSlicer/assets/15314940/f5762a62-bfd3-4faf-a8e2-df5c3780132e

### Actual results

The results in the print is this:

![20240325_185453](https://github.com/SoftFever/OrcaSlicer/assets/15314940/5f53a9eb-78c0-4999-a1c7-2c81271fbd76)

### Expected results

This bridges should be printed in the Inner/Outer order to be anchored by the other lines!

### Project file & Debug log uploads

[debug_network_Mon_Mar_25_17_44_52.log.zip](https://github.com/SoftFever/OrcaSlicer/files/14750192/debug_network_Mon_Mar_25_17_44_52.log.zip)

### Checklist of files to include

- [X] Log file
- [X] Project file

**Comment by SoftFever** [comment]
Does it happen in 1.9.1 or 2.0.0-beta too?

**Comment by Vitorleoron** [comment]
Yes, worst in the 2.0.0-rc version, but in the overhang wall lines the problem still occur in the old versions too, in the 2.0.0-rc the floating lines happen more often

**Comment by SoftFever** [comment]
> Yes, worst in the 2.0.0-rc version, but in the overhang wall lines the problem still occur in the old versions too, in the 2.0.0-rc the floating lines happen more often

Could you share the project? I'd like to see if there's been any regression. 
Meanwhile, based on the photo, this model has very steep overhangs. 
It's not recommended to use a sandwich or outer wall-first sequence for such models.

**Comment by Dustmuffins** [comment]
> Could you share the project? I'd like to see if there's been any regression. Meanwhile, based on the photo, this model has very steep overhangs. It's not recommended to use a sandwich or outer wall-first sequence for such models.

I'm not the OP, but I was able to recreate it on this file.

I'm not sure if this is the best place to request this, but is it possible to request a feature to switch to inner/outer w/ precise walls if the overhang is this extreme?

[large overhang.zip](https://github.com/SoftFever/OrcaSlicer/files/15141295/large.overhang.zip)

**Comment by tastyratz** [comment]
Does this happen only with Arachne or with classic walls as well?

**Comment by pavlokomarov** [comment]
@SoftFever is any updates? Faced with similar issue: outer wall overhang is printed first, despite Inner/Outer order. Orca slicer 2.1.1

https://github.com/user-attachments/assets/848909a3-0325-44d0-868d-e7b837000f2d

**Comment by FelixBuehler** [comment]
Can confirm this issue. Setting is Inner/Outer, but the overhang wall gets printed first.
![image](https://github.com/user-attachments/assets/0d3dd656-4b5e-4cd8-a50c-d09884749a4b)

**Comment by Maximus-CZ** [comment]
Yo its stale and closed but the issue still persists.. Ideally I want to keep Inner/outer/inner, but overhanging parts should still be printed with inner/outer to prevent printing midair..

**Comment by OsmosisD** [comment]
Someone please reopen and add immunity to staling.

Still happening as of the latest version (2.3.2), especially on 90 degree bridges.

Bridging should be inset *into* the wall by at least one line width for anchoring, but instead the slicer draws an extra perimeter in midair, which doesn't help.

<img width="1893" height="1246" alt="Image" src="https://github.com/user-attachments/assets/52da6020-6e5f-4dfb-bbf3-51c95f1b9e9a" />
